oresat_dxwifi/resources/temperature.py

```diff
@@ -0,0 +1,119 @@
+# Purpose:
+#       Issue: https://github.com/oresat/oresat-dxwifi-software/issues/1.
+#       The temperature "read" by an NTC thermistor is calculated based on the voltage recieved at the ADC input pin.
+#       The calculations are based on https://docs.google.com/document/d/16b_Qd6Pj2IAk5HQQl7JAP1T9Qd6QbKICMd-0swd9L6Y/edit
+#       The resource then sends the calculated temperature to the Object Directory upon resource start. An SDO read callback is also implemented
+
+from olaf.common.resource import Resource
+
+from numpy import log as ln
+from olaf.common.adc import Adc
```

Comment:
- The olaf imports can just be `from olaf import Resource, Adc`
- Use `import math as m` to get log. No reason to add numpy as a dependency.

Reply:
Numpy dependency was removed and the imports have been adjusted to `from Olaf import Resource, Add, logger` and   ##`import math as m`.